minos/cqrs/services.py

```diff
@@ -0,0 +1,32 @@
+"""
+Copyright (C) 2021 Clariteia SL
+This file is part of minos framework.
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+from abc import (
+    ABC,
+)
+
+from dependency_injector.wiring import (
+    Provide,
+)
+
+from minos.common import (
+    MinosConfig,
+    MinosSagaManager,
+)
+
+
+class Service(ABC):
+    """Base Service class"""
+
+    config: MinosConfig = Provide["config"]
+    saga_manager: MinosSagaManager = Provide["saga_manager"]
+
```

Comment:
Hi @vladyslav-fenchak, I'm thinking that it would be a good point to provide the implementation of the `__init__` constructor in order to manually set the `config` and `saga_manager` instances. You can do something similar to the `minos.common.Aggregate` constructor: https://github.com/Clariteia/minos_microservice_common/blob/66bec5f51d0e4f4bdfc376e6436f59b9f609b9df/minos/common/model/declarative/aggregate/model.py#L68-L93

Reply:
Done!